hilti/runtime/include/types/stream.h

```diff
@@ -220,16 +237,16 @@ class Chunk {
         if ( n <= Chunk::SmallBufferSize ) {
             std::array<Byte, Chunk::SmallBufferSize> x{};
             std::copy(ud, ud + n.Ref(), x.data());
-            return Chunk(o, std::move(x), n);
+            return Chunk(o, x, n);
```

Comment:
Once more: wondering about this change?

Reply:
This is related to the signature change in the constructor above which cleans up a clang-tidy warning. I'll move this to its own patch.